test/listing_statuses_test.rb

```diff
@@ -0,0 +1,29 @@
+require "test_helper"
+
+class ListingStatusesTest < EasyBrokerTestBase
+  attr_reader :properties
```

Comment:
I believe this should've been:
```suggestion
  attr_reader :statuses
```
And from`test_search` you can call `statuses` instead of `@statuses`. Or simply remove this `attr_reader`